README.md

```diff
@@ -18,210 +18,294 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
-
-### Prerequisites
-#### Clone the repository
+Darkflame Universe is a server emulator and does not distribute any LEGO® Universe files. A separate game client is required to setup this server emulator and play the game, which we cannot supply. Users are strongly suggested to refer to the safe checksums listed [here](#lego®-universe-11064) to see if a client will work.
+
+## Steps to setup server
+* [Clone this repository](#clone-the-repository)
+* [Install dependencies](#install-dependencies)
+    * [cmake](#cmake)
+		* [Windows and MacOS](#windows-cmake)
+		* [Linux, Ubuntu and others](#linux-ubuntu-or-others-cmake)
+	* [Packages](#packages)
+		* [Linux](#linux-build-packages)
+		* [Windows](#windows-packages)
+		* [MacOS](#macos-build-packages)
+	* [Database](#database)
+		* [Linux](#linux-mariadb)
+		* [Windows](#windows-mariadb)
+		* [MacOS](#macos-mariadb)
+	* [Choosing the right client version for your server](#choosing-the-right-clent-version-for-your-server)
+* [Build the server](#build-the-server)
+	* [Enabling testing](#enabling-testing)
+	* [MacOS](#building-on-macos)
+	* [All other platforms](#building-on-all-other-platforms)
+* [Updating your server](#updating-your-server)
+* [Configuring your server](#configuring-your-server)
+	* [Setting up the client dependencies](#setting-up-the-client-dependencies)
+	* [Setting up the database](#setting-up-the-database)
+	* [Configuration files](#configuration-files)
+* [Verify your setup](#verify-your-setup)
+* [Running the server](#running-the-server)
+	* [First admin user](#first-admin-user)
+	* [Account management tool (Nexus Dashboard)](#account-management-tool-nexus-dashboard)
+	* [Admin levels](#admin-levels)
+* [User Guide](#user-guide)
+	* [Allowing a user to connect to your server](#allowing-a-user-to-connect-to-your-server)
+	* [Brick-by-Brick building](#brick-by-brick-building)
+		* [Allowing a user to build in Brick-by-Brick mode](#allowing-a-user-to-build-in-brick-by-brick-mode)
+* [In-game commands](#in-game-commands)
+* [Credits](#credits)
+	* [Active Contributors](#active-contributors)
+	* [Darkflame Universe Team](#darkflame-universe-team)
+		* [Research and tools](#research-and-tools)
+		* [Community management](#community-management)
+		* [Former contributors](#former-contributors)
+		* [Logo](#logo)
+		* [Special thanks](#special-thanks)
+
+## Clone the repository
+
+If you are on Windows, you may need to download and install git from [here](https://git-scm.com/download/win)
+
+Then run the following command
 ```bash
 git clone --recursive https://github.com/DarkflameUniverse/DarkflameServer
 ```
-#### Python
 
-Some tools utilized to streamline the setup process require Python 3, make sure you have it installed.
+## Install dependencies
 
+### CMake
```

Comment:
CMake is just another package. I'd just include cmake as a package as with the other packages (either as something to install via apt, download on windows, etc) and then add a note under that section to verify the cmake version and that on Linux if it isn't available in your package repos, to eg grab it from kitware